README.md

```diff
@@ -20,6 +20,7 @@ Golang versions currently provided are:
 * [Golang 1.9](1.9)
 * [Golang 1.10](1.10)
 * [Golang 1.11](1.11)
```

Comment:
Please delete version 1.9, 1.10 and 1.11. I would prefer to keep only 1.12.

Reply:
I'd rather have this done in a separate PR?

Reply:
Ah, right, its probably best be done here to avoid merge conflicts. Done